src/ucs/sys/sys.c

```diff
@@ -407,6 +427,12 @@ size_t ucs_get_phys_mem_size()
 
     if (phys_pages == 0) {
         phys_pages = ucs_sysconf(_SC_PHYS_PAGES);
+        if (phys_pages < 0) {
+            phys_pages = 1;
```

Comment:
next time this function is called, it will return 1*page_size, which is not correct. 
Also I've just noticed phys_pages is unsigned type, so it cannot be <0 (where is coverity/compiler??)
anyway this would be correct IMHO:
```
size_t ucs_get_phys_mem_size()
{
    static size_t phys_mem_size = 0;
    long phys_pages;

    if (phys_mem_size == 0) {
        phys_pages = ucs_sysconf(_SC_PHYS_PAGES);
        if (phys_pages < 0) {
            ucs_debug("_SC_PHYS_PAGES is undefined, "
                      "physical memory is considered unlimited");
            phys_mem_size = SIZE_MAX;
        } else {
            phys_mem_size = phys_pages * ucs_get_page_size();
        }
    }
    return phys_mem_size;
}
```